setup.py

```diff
@@ -73,7 +73,7 @@
         "eth-account>=0.5.2,<0.6.0",
         "eth-hash[pycryptodome]>=0.2.0,<1.0.0",
         "eth-typing>=2.0.0,<3.0.0",
-        "eth-utils>=1.9.3,<2.0.0",
+        "eth-utils==1.9.3",
```

Comment:
This will cause problems elsewhere if it remains.  For example Trinity, installs Web3.py and I know that trinity requires a later version of `eth-utils`

Reply:
That's a bummer. eth-utils 1.9.4 breaks web3's linting so will probably have to update that before this can be shipped